Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -522,6 +522,11 @@ private string ResolveStartupFile()
             if (TypeScriptHelpers.IsTypeScriptFile(startupFile))
             {
                 startupFile = TypeScriptHelpers.GetTypeScriptBackedJavaScriptFile(this._project, startupFile);
+                if (startupFile == null)
```

Comment:
I'm concerned that returning `null` affect the scenarios where is an NTVS project that has not defined `TypeScriptOutDir` but the user expect to have a `js` file created on the same place as the `ts` file.

Reply:
Sure, we may want to handle those scenarios, bu today those scenarios throw/crash. So my claim is I'm only making it better :)  In this PR my intention is to gracefully handle the cases where we throw known exceptions today.

Note that the existing code did handle `outDir` being null in cases where we didn't throw.